apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-card.js

```diff
@@ -155,7 +155,9 @@ function CardNavigation( {
 }
 
 function CardOverlayControls( { onMinimize, onDismiss } ) {
-	const buttonClasses = classNames( 'welcome-tour-card__overlay-controls' );
+	const buttonClasses = classNames( 'welcome-tour-card__overlay-controls', {
+		'welcome-tour-card__overlay-controls__absolute': ! isMobile(),
```

Comment:
This should work fine for now but if we need more desktop specific styles, I believe a proper way to do it is to add a `is-desktop` (or a modifier if we are using BEM) class name [at the top level of the wpcom flavoured tour](https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.js#L125).

No action is needed in this PR.
cc: @chriskmnds if you see a potential tour-kit extension for this ✍🏼

Reply:
Yeah, I think there is a bit of a convention mismatch. `__` should really be used to signal a relationship with an element e.g. `foo__title` or `foo__header` (as in the case with `.welcome-tour-card__overlay-controls`). For modifiers, like in the change above, the convention would be to use an `.is-modifier` class and attach it to the parent. See our [CSS/Sass Coding Guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md) for more. I've also seen `--modifier` used elsewhere (I think that's the BEM convention).

Reply:
> I've also seen --modifier used elsewhere (I think that's the BEM convention).

Yes, that's one of the BEM options. You can also use a single underscore as an alternative.
eg: `[block]__[element]_[modifier-name]_[modifier-value]`

But I think I'd go with our documented approach using `.is-modifier` as it doesn't bloat the code too much 😬